met-web/src/components/engagement/form/AddSurveyBlock.tsx

```diff
@@ -95,6 +135,20 @@ export const AddSurveyBlock = () => {
                     spacing={2}
                     sx={{ padding: '1em' }}
                 >
+                    <Grid item xs={12}>
+                        <Box sx={{ width: '100%', typography: 'body1' }}>
+                            <MetTabs tabs={tabs} updateTabName={updateTabName} />
+                        </Box>
+                        <If condition={selectedTabName == SubmissionStatus[SubmissionStatus.Upcoming]}>
```

Comment:
We shouldn't handle the conditional rendering of the tabs. The MUI TabContext and TabPanel components should handle that for us 
![image](https://user-images.githubusercontent.com/91914654/208715321-e8a4a182-d3fa-4a5f-8c9b-7bed58c5bf61.png)

Reply:
Changed the logic to have separate code for survey block tabs